include/support.h

```diff
@@ -330,4 +330,23 @@ template <typename T>
 std::pair<std::unique_ptr<T[]>, T*> make_unique_aligned_array(
         const size_t byte_alignment, const size_t req_elems, const T& init_val = {});
 
+// Allow for overload resolition for use with std::visit in combination with an
+// std::variant. A list of functors for each type must be specified.
```

Comment:
Maybe include the comment from the commit message here? Not everybody knows what a "functor" is, I think, so this is a bit cryptic.